packages/core/src/lib/implementation/read-code-pushup-config.ts

```diff
@@ -9,15 +13,16 @@ export class ConfigPathError extends Error {
 }
 
 export async function readCodePushupConfig(filepath: string) {
-  const isFile = (await stat(filepath)).isFile();
+  const validFilepath = filePathSchema('config file path').parse(filepath);
```

Comment:
I would revert these changes. We don't need to include Zod just for this, the "`filePathSchema`" only checks string length. Just check if the file exists with `stat`, we don't need to complicate things.

Reply:
done